doc/source/whatsnew/v0.15.0.rst

```diff
@@ -86,7 +81,10 @@ For full docs, see the :ref:`categorical introduction <categorical>` and the
     df["grade"].cat.categories = ["very good", "good", "very bad"]
 
     # Reorder the categories and simultaneously add the missing categories
-    df["grade"] = df["grade"].cat.set_categories(["very bad", "bad", "medium", "good", "very good"])
+    df["grade"] = df["grade"].cat.set_categories(["very bad",
+                                                  "bad", "medium",
+                                                 "good", "very good"]
```

Comment:
this indentation needs to be at the same level as the previos. The `)` in the next line should be at the end of this.